file-availability.md

```diff
@@ -1,61 +1,62 @@
 ---
 highlighter: none
 layout: file_avail
-title: File Availability Options
+title: Small Input and Output File Transfer
 ---
 
-
-HTCondor File Transfer
-======================
-
-HTCondor file transfer is the standard solution for file portability,
-and is built in to HTCondor job scheduling. You can see HTCondor file
-transfer as introduced in our \"Intro to Running HTCondor Jobs\" and in
-the [HTCondor
-Manual.](http://research.cs.wisc.edu/htcondor/manual/v7.8/2_5Submitting_Job.html#SECTION00354000000000000000)
-
-Contents
+Table of Contents
 --------
 
-1.  [Applicability](#Appli)
-2.  [Transferring Input Files](#input)
-3.  [Transferring Output Files](#output)
+[HTCondor File Transfer](#transfer)   
+[Applicability](#Appli)    
+[Transferring Input Files](#input)   
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Important Considerations](#important-considerations)   
```

Comment:
are these `&nbsp;` tab characters or spaces?

Reply:
spaces, it will indent these lines by 6 spaces